### DISCUSSION TITLE
Diffs have seemingly unchanged content

### DISCUSSION THREAD

**Discussion by beyondcompute** [renovatebot/renovate#16807] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

This diff seemingly contains files without meaningful changes: https://app.renovatebot.com/package-diff?name=tsc-watch&from=4.6.2&to=5.0.3

I am not sure if it’s an actual “bug” or just line endings format or something have changed there. Anyway, thank you!

Screenshot of what I’m seeing just in case it does not reproduce anymore:
<img width="644" alt="" src="https://user-images.githubusercontent.com/248055/181252591-145b226a-c4c6-4f4c-8855-3d2b26166498.png">

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
There is no bug here and nothing to discuss.

**Comment by beyondcompute** [comment, marked as answer]
Ok, the issue was with the line endings indeed: https://github.com/gilamran/tsc-watch/blob/master/CHANGELOG.md#v503---07042022

Still the diff is harder to comprehend in this situation. So the whitespace-ignoring option could have been helpful.
